Mathlib/Analysis/SpecialFunctions/Integrals.lean

```diff
@@ -255,7 +255,7 @@ theorem intervalIntegrable_inv_one_add_sq :
 /-! ### Integrals of the form `c * ∫ x in a..b, f (c * x + d)` -/
 
 
--- Porting note (#10618): was @[simp];
+-- Porting note (#10618): was @[simp]
 -- simpNF says LHS does not simplify when applying lemma on itself
```

Comment:
Did you check that this is still applicable?

Reply:
I had not. I just pushed a commit adding back @[simp]: at least #lint passes; let's see if CI has anything else to say.

Reply:
(If successful, that change will be a separate PR.)

Reply:
And linting passed :tada: preparing a PR now

Reply:
@grunweg did you know you can run `#lint` right after the decl? It's not a global test, but it would probably have told you whether this particular issue was still a problem or not.

Reply:
Yes, I do, thanks for asking!
#15015 re-simps this lemmas and a few more